sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp/Azure.Monitor.OpenTelemetry.Exporter.AotCompatibilityTestApp.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net7.0;net6.0;net462</TargetFrameworks>
```

Comment:
Why target `net462` trimming or AOT is not happening on .NET Framework.

Reply:
These are required framework for projects in a "test" directory in this repo.
I also tried putting this app in a separate "aot" directory but that several other requirements.